app/routes/circulars/index.tsx

```diff
@@ -23,6 +23,7 @@ import {
   TextInput,
 } from '@trussworks/react-uswds'
 import classNames from 'classnames'
+import { clamp } from 'lodash'
```

Comment:
Load just the function that we need, to avoid bundling all of lodash.

```suggestion
import clamp from 'lodash/clamp'
```